src/trans.rs

```diff
@@ -72,6 +107,10 @@ impl<'v, 'tcx: 'v> Drop for BinaryenModuleCtxt<'v, 'tcx> {
     }
 }
 
+// The address in wasm linear memory where we store the stack pointer
+// TODO: investigate where should the preferred location be
+const STACK_POINTER_ADDRESS : i32 = 0;
```

Comment:
yeah true. I just put it in there to try if it worked and it did, but as Alon mentioned it should be, say, at 8 :)